generator/uscore/uscore_generator.rb

```diff
@@ -938,12 +938,10 @@ def create_multiple_or_test(sequence)
 
               reply = get_resource_by_params(versioned_resource_class('#{sequence[:resource]}'), search_params)
               validate_search_reply(versioned_resource_class('#{sequence[:resource]}'), reply, search_params)
-              assert_response_ok(reply)
               resources_returned = fetch_all_bundled_resources(reply, check_for_data_absent_reasons)
-
-
+              applicable_resources_returned = resources_returned.reject { |entry| entry.class == FHIR::OperationOutcome }
```

Comment:
Could server returns resources other than `sequence[:resource]` and `OperationOutcome`?
I think it would be safer to filter in resource that matches `sequence[:resource]`

Reply:
I proactively answered this question in one of my PR comments. The test would have failed already if any other resource type was in the response based on the logic in `validate_search_reply`.